ui/imports/shared/status/StatusChatInputReplyArea.qml

```diff
@@ -73,16 +76,16 @@ Rectangle {
         }
     }
 
-    StatusChatImage {
+    StatusMessageImageAlbum {
         id: imageThumbnail
         anchors.left: replyToUsername.left
         anchors.top: replyToUsername.bottom
         anchors.topMargin: 2
-        imageWidth: 64
-        imageSource: root.image
-        chatHorizontalPadding: 0
-        visible: root.contentType === Constants.messageContentType.imageType
-        playing: false
+        album: root.albumCount > 0 ? root.album : [root.image]
+        albumCount: root.albumCount > 0 ? root.albumCount : 1
```

Comment:
```suggestion
        albumCount: root.albumCount || 1
```

Reply:
its the same but I think its more readable